Dalamud/Utility/EventHandlerExtensions.cs

```diff
@@ -99,6 +100,23 @@ public static void InvokeSafely(this IFramework.OnUpdateDelegate? updateDelegate
         }
     }
 
+    /// <summary>
+    /// Replacement for Invoke() on OnMenuOpenedDelegate to catch exceptions that stop event propagation in case
+    /// of a thrown Exception inside of an invocation.
+    /// </summary>
+    /// <param name="openedDelegate">The OnMenuOpenedDelegate in question.</param>
+    /// <param name="argument">Templated argument for Action.</param>
+    public static void InvokeSafely(this IContextMenu.OnMenuOpenedDelegate? openedDelegate, MenuOpenedArgs argument)
```

Comment:
(non-blocking) this is going to become a headache at some point... need to think of a better strategy for this.